x11-plugins/wmSun/wmSun-1.05-r1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit toolchain-funcs
+
+MY_P=${P/S/s}
+DESCRIPTION="dockapp which displays the rise/set time of the sun"
+HOMEPAGE="https://www.dockapps.net/wmsun"
+SRC_URI="https://dev.gentoo.org/~voyageur/distfiles/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~sparc ~x86"
+
+RDEPEND=">=x11-libs/libdockapp-0.7:=
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXpm"
+DEPEND="${RDEPEND}
+	x11-base/xorg-proto"
+
+S="${WORKDIR}/${MY_P}"
+
+src_compile() {
+	emake CC="$(tc-getCC)" CFLAGS="${CFLAGS}"
+}
+
+src_install() {
+	emake DESTDIR="${D}" PREFIX=/usr install
+	dodoc README {BUGS,TODO}
```

Comment:
You can also set DOCS variable and call einstalldocs here

(note this is just an alternative way, dodoc direct call still works fine of course)